tests/unit/__fixtures__/fileActions.js

```diff
@@ -0,0 +1,174 @@
+const meta = {
```

Comment:
Please move into `web-app-files/tests/__fixtures__` (not into the unit subfolder)